mmv1/third_party/terraform/resources/resource_storage_transfer_job.go

```diff
@@ -329,6 +329,13 @@ func gcsDataSchema() *schema.Resource {
 				Type:        schema.TypeString,
 				Description: `Google Cloud Storage bucket name.`,
 			},
+			"path": {
+				Optional:     true,
+				Computed:     true,
+				Type:         schema.TypeString,
+				Description:  `Google Cloud Storage path in bucket to transfer`,
+				ValidateFunc: validateRegexp("^[^/].*/$"),
```

Comment:
Is this a valid regex? Do the `/` characters not need to be escaped? What is the goal here

Reply:
Specifically, this looks like it's trying to forbid a string that starts with `/`. But the description says "As such, it should generally not begin with a '/'." which feels like a suggestion rather than a restriction

Reply:
I removed the validation function